src/worker/core/player/genRatings.basketball.ts

```diff
@@ -183,6 +183,35 @@ const genRatings = (
 		skills: [],
 	};
 
+	// If the youngest players generated aren't 19, scale ratings to match age
+	const age = g.get("draftAge")[0];
+	if (age !== 19) {
+		// Youngest prospects != 19 will be scaled, scaling stops at age 14 and 28
```

Comment:
Can you take another look at all these?

- Basketball is currently scaled from 12 to 28, not 14 to 28
- Football is currently scaled from 14 to 30, not 14 to 28
- Hockey is currently scaled just like basketball, but I'm not sure if that's intended because in the other two sports you use the same age in the `scale` formula as in the `if` right before it. If it's intended for those two numbers to always be the same, use a constant like `const DEFAULT_MIN_AGE = 19;` (different in each file). That prevents them from ever getting out of sync.

If the goal was to have all the age ranges the same for every sport, like the comments suggest, I recommend you put the `helpers.bound` around `age`, not around the entire `scale` formula. Then it'll be much more clear what's going on.

Reply:
good eye, give me a minute

Reply:
Ok so some reasoning behind these numbers if you aren't sure. I set the min max age scaling limits to prevent super leagues (So not every prospect is a 90 OVR demigod for 35 year old DP's) and to prevent trash prospects with a rating of 0 in every single rating. So hitting the max scaling will produce a class that looks like a typical league (few studs, lot of role players), and the min scaling will at least give some promising players instead of everyone being absolute trash.